app/views/sessions/new.html.erb

```diff
@@ -4,4 +4,5 @@
 <%= f.email_field :email, placeholder: "Email" %>
 <%= f.password_field :password, placeholder: "Password" %>
 <%= f.submit "Login" %>
+<class ="button"><%= link_to "Forgot Password", password_path %></>
```

Comment:
It looks like there might be an error on this line? Was the `</>` supposed to be a `</br>`?